### PR TITLE
Bug 1883701: Bump openshift/api

### PR DESCRIPTION
**Pull request by dmage**
This PR vendors new openshift/api to get new CRDs.

### CODE REVIEW

File: manifests/00-custom-resource-definition.yaml [hunk 1]
```diff
@@ -10,1028 +11,1024 @@ spec:
     listKind: IngressControllerList
     plural: ingresscontrollers
     singular: ingresscontroller
-  preserveUnknownFields: false
-  scope: ""
-  subresources:
-    scale:
-      labelSelectorPath: .status.selector
-      specReplicasPath: .spec.replicas
-      statusReplicasPath: .status.availableReplicas
```
Comment (Miciah) [inline manifests/00-custom-resource-definition.yaml:19]: Where did this go?

**Comment by Miciah** [comment]
> [openshift/api#736](https://github.com/openshift/api/pull/736) added restrictions for logLevel values. This PR vendors new openshift/api to get new CRDs.

The `IngressController` API doesn't have any `LogLevel` field, so this PR is not needed on that account.  That said, it would be good to bump to get the v1 CRD.  We need to address https://github.com/openshift/cluster-ingress-operator/pull/469/files#r496931106 though.

File: manifests/00-custom-resource-definition.yaml (hunk 1 above)
Comment (dmage) [reply manifests/00-custom-resource-definition.yaml:19]: It seems it was removed at https://github.com/openshift/api/pull/720

Comment (sttts) [reply manifests/00-custom-resource-definition.yaml:19]: tagged the PR.

Comment (dmage) [reply manifests/00-custom-resource-definition.yaml:19]: bumped openshift/api again

**Comment by Miciah** [comment]
Thanks!
/approve

**Pull request merged**